### PR TITLE
Fixes a few runtimes

### PR DESCRIPTION
**Pull request by Hinaichigo**
This chips away at a few of the simpler runtimes that I could see on the logs.

### CODE REVIEW

File: code/datums/emotes.dm [hunk 1]
```diff
@@ -70,7 +70,7 @@
 			if(!M.client || isnewplayer(M))
 				continue
 			var/T = get_turf(user)
-			if(isobserver(M) && M.client && (M.client.prefs.toggles & CHAT_GHOSTSIGHT) && !(M in viewers(T)))
+			if(isobserver(M) && M.client && (M.client.prefs?.toggles & CHAT_GHOSTSIGHT) && !(M in viewers(T)))
```
Comment (DamianX) [inline code/datums/emotes.dm]: Under what circumstances can we expect prefs to be `null`?

Comment (Hinaichigo) [reply code/datums/emotes.dm]: I'm not sure, it occurs here:
```
[23:59:09] Runtime in emotes.dm,73: Cannot read null.toggles
  proc name: run emote (/datum/emote/proc/run_emote)
  usr: Marisa Miazga ([...]) (/mob/living/carbon/human)
  usr.loc: The floor (165, 271, 1) (/turf/simulated/floor/shuttle)
  src: /datum/emote/living/carbon/sou... (/datum/emote/living/carbon/sound/scream)
  call stack:
  /datum/emote/living/carbon/sou... (/datum/emote/living/carbon/sound/scream): run emote(Crush Martins (/mob/living/carbon/human), 1, null, 1, null)
  /datum/emote/living/carbon/sou... (/datum/emote/living/carbon/sound/scream): run emote(Crush Martins (/mob/living/carbon/human), 1, null, 1, null)
  Crush Martins (/mob/living/carbon/human): emote("screams", null, 1, 1, null)
  Crush Martins (/mob/living/carbon/human): audible scream(null)
  the electrode (/obj/item/projectile/energy/electrode): hit apply(Crush Martins (/mob/living/carbon/human), null)
  the electrode (/obj/item/projectile/energy/electrode): on hit(Crush Martins (/mob/living/carbon/human), 10)
  Crush Martins (/mob/living/carbon/human): bullet act(the electrode (/obj/item/projectile/energy/electrode), "chest")
  Crush Martins (/mob/living/carbon/human): bullet act(the electrode (/obj/item/projectile/energy/electrode), "chest")
  the electrode (/obj/item/projectile/energy/electrode): to bump(Crush Martins (/mob/living/carbon/human))
  the electrode (/obj/item/projectile/energy/electrode): perform bump()
  the electrode (/obj/item/projectile/energy/electrode): Move(the floor (173,272,1) (/turf/simulated/floor/shuttle), 0, 0, 0, 0)
  the electrode (/obj/item/projectile/energy/electrode): bresenham step(4, 1, 4, 1)
  the electrode (/obj/item/projectile/energy/electrode): process step()
  the electrode (/obj/item/projectile/energy/electrode): process()
```
So they need to be in `dead_mobs_list`, have a client, but not have prefs.

Comment (Hinaichigo) [reply code/datums/emotes.dm]: So maybe it'd be better to not use `?.` here and elsewhere, and instead stop the calls to the various procs in advance?

Comment (Hinaichigo) [reply code/datums/emotes.dm]: I simplified it a bit.
The only places I could find where `prefs = null` occurs is in the definition of the client datum, and `datum/preferences_subsection/Destroy()`.
Maybe the player is being given a new body at some point but prefs isn't transferred properly, or another runtime is occuring elsewhere and it's being fed in.

A similar runtime happens in `say_dead()`

Comment (DamianX) [reply code/datums/emotes.dm]: I don't know. It doesn't look like `prefs` is ever supposed to be null. I would be hesitant to start adding checks for that possibility everywhere it's used, we should figure out the root cause.

Comment (Hinaichigo) [reply code/datums/emotes.dm]: That makes sense.
`prefs` being `null` occurs a lot in [http://game.ss13.moe/logs/2022/runtime/2022-05-28-runtime.log](http://game.ss13.moe/logs/2022/runtime/2022-05-28-runtime.log)... a lot of times with checking space parallax prefs.

**Pull request closed without merging**